File: backend/services/course-service/src/main/java/com/lmspilot/course/CourseServiceApplication.java
package com.lmspilot.course;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;


@SpringBootApplication(scanBasePackages = "com.lmspilot")
public class CourseServiceApplication {
    public static void main(String[] args) { SpringApplication.run(CourseServiceApplication.class, args); }
}

File: backend/services/course-service/src/main/kotlin/com/lmspilot/course/CourseServiceApplication.kt
package com.lmspilot.course

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication(scanBasePackages = ["com.lmspilot"])
class CourseServiceApplication

fun main(args: Array<String>) {
    runApplication<CourseServiceApplication>(*args)
}
